### ISSUE TITLE
searching / clustering by collector names  ¿splitting? ¿standardizing name format?

### ISSUE THREAD

**Issue by abubelinha** [gbif/pipelines#782]
I have a question (related to this [portal usage question](https://github.com/gbif/portal-feedback/issues/2171#issuecomment-1239271332)) and also somehow to this [clustering question](https://github.com/gbif/pipelines/issues/777):

When GBIF compares collector names for clustering, do you first split them according to `recordedBy` suggested `|` separator?
I mean, if recordedBy contains "Person A | Person B", do you compare the whole string or do you treat "Person A" and "Person B" somehow individually?

The latter might sound odd, but when you want to cluster by means of `recordNumber`, in most cases that number is referred only to one person in the list (usually the first one, or at least that's the recommendation).

So, in the above linked example, these are the values:

| dwc field | value |
| -- | -- |
| Record number | 2220 |
| Recorded by | Bautista, H.P.; Oubiña, J. |

Actually, 2220 is referred only to H.P.Bautista, so the splitting question above may have more sense (specially because sometimes the companion collectors may not be cited -or not all of them when it's a big group-, and that would break clusters based on comparisons of the whole string). 

BTW, in some datasets people are cited in the opposite way (initials + surname):
H.P. Bautista; J. Oubiña

But if GBIF splits recordedBy string into different authors, perhaps GBIF could try to standardize them prior to clustering or answering portal searches.

Thanks
@abubelinha

**Comment by timrobertson100** [comment]
> When GBIF compares collector names for clustering, do you first split them according to recordedBy suggested | separator?

Yes, we split on `|` into names [as data are ingested](https://github.com/gbif/pipelines/blob/dev/sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/BasicInterpreter.java#L294) and indexed for search. The clustering comes later in our processing and  makes use of those splits looking for [overlap of any person](https://github.com/gbif/pipelines/blob/0498f9d05c2ec1c007871ec5662bb7dae225d0cf/sdks/core/src/main/java/org/gbif/pipelines/core/parsers/clustering/OccurrenceRelationships.java#L186) 

We have recently merged codebases with the ALA in this project, which still has some areas to fully align, and I see they have [code for normalising names](https://github.com/gbif/pipelines/blob/dev/livingatlas/pipelines/src/main/java/au/org/ala/pipelines/parser/CollectorNameParser.java) which I will explore for the clustering
